extensions/smallrye-stork/deployment/pom.xml

```diff
@@ -50,7 +60,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
```

Comment:
Could we disable the tests using this feature instead of everything?

Because by doing so, we lose all testing on Windows.